result/src/commonMain/kotlin/com/github/kittinunf/result/Result.kt

```diff
@@ -3,6 +3,10 @@ package com.github.kittinunf.result
 import com.github.kittinunf.result.Kind.Failure
 import com.github.kittinunf.result.Kind.Success
 
+fun <V: Any> V.success(): Result.Success<V> = Result.success(this)
```

Comment:
Oh btw, one last question. This should be constraint over `V?` right?

Reply:
Nice catch! Here it is: b3e0576dab4201ec724b7a5958a4edb2be0716a7